<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTimMLDockWidgetBase</class>
 <widget class="QDockWidget" name="QgisTimDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTimML</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Elements">
       <attribute name="title">
        <string>Dataset</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>10</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Model">
       <attribute name="title">
        <string>Elements</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>13</y>
          <width>109</width>
          <height>415</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="aquiferButton">
           <property name="text">
            <string>Aquifer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="constantButton">
           <property name="text">
            <string>Constant</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="wellButton">
           <property name="text">
            <string>Well</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="headWellButton">
           <property name="text">
            <string>HeadWell</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="circularAreaSinkButton">
           <property name="text">
            <string>CircularAreaSink</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="uniformFlowButton">
           <property name="text">
            <string>UniformFlow</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="headLineSinkButton">
           <property name="text">
            <string>HeadLineSink</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="lineSinkDitchButton">
           <property name="text">
            <string>LineSinkDitch</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="impLineDoubletButton">
           <property name="text">
            <string>ImpLineDoublet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="leakyLineDoubletButton">
           <property name="text">
            <string>LeakyLineDoublet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="polygonInhomButton">
           <property name="text">
            <string>PolygonInhom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="domainButton">
           <property name="text">
            <string>Domain</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
